### ISSUE TITLE
Clearer error message for missing file that is listed in .include

### ISSUE THREAD

**Issue by akrabat** [nimbella/nimbella-cli#5]
If a file listed in `.include` cannot be found, then the error message is:

```
Error: ENOENT: no such file or directory, lstat 
  '/Users/rob/Projects/nimbella/nim-todo/packages/todo/list/list.php'
```

it would be helpful if the message indicated that this file was expected _because_ it is listed in `.include`.

**Comment by joshuaauerbachwatson** [comment]
There has always been a friendlier message; it was being bypassed because a lower level function that should not throw was throwing.  Fixed in the source.  The fix will be in the next stable release of `nim`.

**Comment by joshuaauerbachwatson** [comment]
The fix for this has been in stable release 1.7.0 of `nim` since 9/1.  I neglected to close out this issue at the time.

**Comment by akrabat** [comment]
Thanks!
